README.md

```diff
@@ -6,6 +6,10 @@ A list of active declarative programming streams:
   - Speaking: EN
   - Programming Languages: **Haskell**, Purescript, Unison
   - Schedule: T/R 7:30-10:30 PM EST
+- [gillchristian](gillchristian):
```

Comment:
The `[...]` should contain the twitch link to your channel

Reply:
You are totally right about both comments. I made the PR from the phone so i missed a few things 😅 fixed now